File: cpp/FizzBuzz.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
	for (int i = 1; (i & 0x0FF) <= 100; ++i &= 0x0FF){
		cout << (!((i & 0x0FF) % 3) ? (i |= 0x100), "Fizz" : "") << (!((i & 0x0FF) % 5) ? (i |= 0x100), "Buzz" : ""), ((i & 0x100) ? cout << "" : cout<< i), cout << "\n"; 
	}
}

File: python/citi/fizzbuzz.py
print '\n'.join(['Fizz'*(x % 3 == 0) + 'Buzz'*(x % 5 == 0) or str( x ) for x in xrange( 1,101 )])

"""
Write a program that prints the numbers from 1 to 100. 
But for multiples of three print “Fizz” instead of the number and for the multiples of five print “Buzz”. 
For numbers which are multiples of both three and five print “FizzBuzz”.
"""
